package.json

```diff
@@ -10,8 +10,11 @@
     "build:run": "nodemon app.js",
     "build": "npm run clean; NODE_ENV=production webpack --color --config webpack.prod.js",
     "clean": "rimraf build",
-    "docker:debug": "docker-compose -f docker-compose.debug.yml -p example_webapp_debug up",
-    "docker:prod": "docker-compose -f docker-compose.prod.yml -p example_webapp_prod up"
+    "dev": "npm run build:watch & npm run build:run",
+    "docker:clean": "docker image prune",
+    "docker:debug": "docker-compose -f docker-compose.debug.yml -p example_webapp_debug up --build",
+    "docker:prod": "docker-compose -f docker-compose.prod.yml -p example_webapp_prod up --build",
+    "docker:test": "docker-compose -f docker-compose.test.yml -p example_webapp_test up --build"
```

Comment:
On the three compose instructions: --build rebuilds the image (using the cache ofc) which allows you to develop faster bc it responds to any source code/config/dependency changes without you having to manually rebuild the images.

This leaves a bunch of hollow image aliases though bc it doesnt delete the old ones when you rebuild. Docker image prune will delete the deprecated images to clean up space.